core/src/test/scala/akka/persistence/cassandra/query/EventsByTagSpec.scala

```diff
@@ -17,7 +17,9 @@ import akka.persistence.query.scaladsl.{ CurrentEventsByTagQuery, EventsByTagQue
 import akka.persistence.query.{ EventEnvelope, NoOffset, TimeBasedUUID }
 import akka.persistence.{ PersistentActor, PersistentRepr }
 import akka.serialization.SerializationExtension
+import akka.stream.testkit.TestSubscriber
```

Comment:
Most of this file's changes are shutting down the streams if there is a failure so the logs are spammed for the rest of the run, i'll mark out important changes